components/taupage.rst

```diff
@@ -542,20 +543,64 @@ And the Agent will follow these logs:
 
 You can get your Account Key from the Logentries Webinterface under /Account/Profile
 
-
 scalyr_account_key
 ------------------
 
 **(optional)**
 
+Options: see :ref:`scalyr_account_key <scalyr_account_key>` in :ref:`logging <logging>`
```

Comment:
Should we mark the usage of these options outside of the `logging` section as deprecated?

Reply:
Marked old scalyr options in main yaml config branch as deprecated.